src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import {Movie} from "../../types/movie";
+
+type Props = {
+  movie: Movie;
+}
+
+
+export const MovieCard: React.FC<Props> = ({movie}) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={movie.imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src={movie.imdbUrl} alt="imdb" />
```

Comment:
```suggestion
            <img src="images/imdb-logo.jpeg" alt="imdb" />
```
я не зовсім впевнений але мабуть саме через це в тебе не відображається логотип  IMDB